### PR TITLE
Remove copy text indent

### PR DESCRIPTION
**Pull request by dgroddick**
To remove the whitespace that adds extra indentation in src/pages/debug-data.php
issue https://github.com/WordPress/health-check/issues/176

### CODE REVIEW

File: src/pages/debug-data.php [hunk 1]
```diff
@@ -38,8 +38,7 @@
 			}
 			?>
 			<div id="system-information-english-copy-wrapper" style="display: none;">
-					<textarea id="system-information-english-copy-field" class="widefat" rows="10">`
-						<?php
+					<textarea id="system-information-english-copy-field" class="widefat" rows="10">`<?php
```
Comment (Clorith) [inline src/pages/debug-data.php]: The PHP opening tag should be on the next line down to both conform with the coding standards, and also be properly parseable by the WordPress.org forums as codetags. Removing the indentations before it on a newline should have the same desired effect.

Comment (Clorith) [inline src/pages/debug-data.php]: The PHP opening tag should be on the next line down to both conform with the coding standards, and also be properly parseable by the WordPress.org forums as codetags. Removing the indentations before it on a newline should have the same desired effect.

Comment (dgroddick) [reply src/pages/debug-data.php]: I've put the php tag back on the newline and removed the indentation.

**Comment by Clorith** [comment]
Thanks for the PR, and the PHPCS cleanup!

I see that it re-introduced the indentation when it was cleaned up, likely to pass PHPCS checks. I think you can safely remove the indentation for the opening PHP tags here, and utilize the `// @codingStandardsIgnoreLine` inline comment to skip the check on that particular line. That'll uphold the phpcs checks overall, but still let you fix the inentations.

**Comment by dgroddick** [comment]
Ignoring line 42 seems to work, but 94 breaks the code block below it down to line 133. I can remove all the phpcs errors by removing the indentation of the entire block, but I'm not sure that's the right way to go.
Build still fails.

screenshot from 2018-10-07 13-33-57

**Comment by Clorith** [comment]
Hmm, would a wrapper function help here, a function that we can pass the `$info` array holder into (that way it'll work for any language we use, so it can be reused). That means we don't need to worry about the indents as it's just a `<textarea><?php functioncall( $info ); ?></textarea>`?

This would take care of the invalid first line indentation, and also allow us to more easily make visual changes in the future.

**Comment by Clorith** [comment]
Fixed in an iteration of this in #211

**Pull request closed without merging**